performance-tests/gradle.properties

```diff
@@ -11,7 +11,11 @@ mlUsername=test-data-hub-developer
 mlPassword=password
 
 # Parameters for mapping functions test
-sourceDocumentCount=1000
+## sourceDocumentCount=1000
```

Comment:
This should be in a section of parameters that apply to multiple tests, as it still applies to the mapping functions test